doc/quickstart.md

```diff
@@ -0,0 +1,221 @@
+
+ ## Installation (see [README](https://github.com/pyccel/pyccel/blob/master/README.rst) file)
+
+ ## Command line usage
+
+-   Open a terminal app, iterm or terminal for MacOs, terminal for Linux.
+
+-   After the installation, type `pyccel`, the usage should be shown, If this is the case then the installation has succeeded.
+
+-   Create a Python file that contains simple lines of code to see what will happen.
+    -   To create the file `touch file_name.py`.
+    -   Use your favorite text editor to fill the file with some lines of code or just type `echo 'print("hello, world!\n")' > file_name.py` for a quick test, `cat file_name.py` to make sure that your lines in the file.
+    -   To generate the C/Fortran code form your Python code, type `pyccel file_name.py` or `pyccel file_name.py --language fortran` to generate Fortran code, and `pyccel file_name.py --language c` to generate C code.
+    -   No problems ? You should discover `__pyccel__` the directory that contains your generated code and some other stuff.
+
+E.g (using `@types` decorator/python type hints and a recursive function with a typed return), To specify the types of the function arguments and its return, we need to import the `@types` decorator from pyccel.decorators (as you can see in `file_name.py` first line) and then specify the types for each function argument in `@types` using the following:
+-   The syntax for the decorator is: `@types('1stArgType', '2ndArgType', 'NthArgType', results='return_type')`, or to declare arrays: `@types('1stArgType[:]', '2ndArgType[:,:]', 'NthArgType[dimensions]', results='return_type')`, The expression `[:]` means that the array has 1 dimension. 2 dimensions would be specified with `[:,:]`. The number of dimensions of an array is equal to the number of comma-separated colons in the square brackets. So `arr[:,:,:]` means that the array `arr` has 3 dimensions and so on.
+-   In the function we just use python syntax `def fun(1stArg, 2ndArg, NthArg)`.
+-   Also, You can specify the function arguments types using python type hints, `def fun(1stArg: 1stArgType, 2ndArg: 2ndArgType, NthArg: NthArgType) -> returnType`
+   
+In `@types` decorator, pyccel supports the following data types: real, double, float, pythonfloat, float32, float64, pythoncomplex, complex, complex64, complex128, int8, int16, int32, int64, int, pythonint, integer, bool, pythonbool.
```

Comment:
why do we have `pythonint`? int is sufficient the same for `integer` and `pythonbool`.